src/helpers/arrow.hpp

```diff
@@ -35,26 +35,27 @@ constexpr std::size_t arrow_bytes_alignment_k = 64;
 namespace arf = arrow::flight;
 namespace ar = arrow;
 
-inline static std::string const kFlightListCols = "list_collections";    /// `DoGet`
-inline static std::string const kFlightSample = "sample";                /// `DoGet`
-inline static std::string const kFlightColCreate = "create_collection";  /// `DoAction`
-inline static std::string const kFlightColDrop = "remove_collection";    /// `DoAction`
-
-inline static std::string const kFlightListSnap = "list_snapshots";      /// `DoGet`
-inline static std::string const kFlightSnapCreate = "create_snapshot";   /// `DoAction`
-inline static std::string const kFlightSnapExport = "export_snapshot";   /// `DoAction`
-inline static std::string const kFlightSnapDrop = "remove_snapshot";     /// `DoAction`
-
-inline static std::string const kFlightTxnBegin = "begin_transaction";   /// `DoAction`
-inline static std::string const kFlightTxnCommit = "commit_transaction"; /// `DoAction`
-
-inline static std::string const kFlightWrite = "write";                  /// `DoPut`
-inline static std::string const kFlightRead = "read";                    /// `DoExchange`
-inline static std::string const kFlightWritePath = "write_path";         /// `DoPut`
-inline static std::string const kFlightMatchPath = "match_path";         /// `DoExchange`
-inline static std::string const kFlightReadPath = "read_path";           /// `DoExchange`
-inline static std::string const kFlightScan = "scan";                    /// `DoExchange`
-inline static std::string const kFlightMeasure = "measure";              /// `DoExchange`
+inline static std::string const kFlightRetrieveMetadata = "retrieve_metadata"; /// `DoGet`
+inline static std::string const kFlightListCols = "list_collections";          /// `DoGet`
+inline static std::string const kFlightSample = "sample";                      /// `DoGet`
+inline static std::string const kFlightColCreate = "create_collection";        /// `DoAction`
+inline static std::string const kFlightColDrop = "remove_collection";          /// `DoAction`
+
+inline static std::string const kFlightListSnap = "list_snapshots";            /// `DoGet`
```

Comment:
Not sure if I can see the changes in those lines. Are there any?

Reply:
No it is just a formatting

Reply:
I added this line 
` inline static std::string const kFlightRetrieveMetadata = "retrieve_metadata";`
And the formatter arranged comments in one line